PolyEngine/RenderingDevice/OpenGL/Src/Pipeline/EnvCapture.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <Defines.hpp>
+#include "Common/GLUtils.hpp"
+#include "Proxy/GLShaderProgram.hpp"
+
+namespace Poly
+{
+	class SkyboxWorldComponent;
+	class GLRenderingDevice;;
+	class GLShaderProgram;
+
+	class EnvCapture : public BaseObject<>
```

Comment:
Do i understand correctly that this will provide ability for static gemoetry world reflections?